test/integration/055_ref_override_test/macros/ref_override_macro.sql

```diff
@@ -0,0 +1,2 @@
+-- Macro to override ref and always return the same result
+{% macro ref(modelname) %} seed_2 {% endmacro %}

```

Comment:
Here's a suggestion for a perhaps more interesting macro that exercises the code more completely:

```suggestion
{% macro ref(modelname) %}
{% do return(builtins.ref(modelname).replace_path(identifier='seed_2')) %}
{% endmacro %}

```